beacon-chain/sync/receive_block.go

```diff
@@ -88,36 +99,30 @@ func (rs *RegularSync) validateAndProcessBlock(ctx context.Context, blockMsg p2p
 	blockRoot, err := hashutil.HashBeaconBlock(block)
 	if err != nil {
 		log.Errorf("Could not hash received block: %v", err)
-		return [32]byte{}, false, err
+		return nil, nil, false, err
 	}
 
 	log.Debugf("Processing response to block request: %#x", blockRoot)
 	hasBlock := rs.db.HasBlock(blockRoot)
-	span.AddAttributes(trace.BoolAttribute("hasBlock", hasBlock))
 	if hasBlock {
 		log.Debug("Received a block that already exists. Exiting...")
-		return [32]byte{}, false, nil
+		return nil, nil, false, err
 	}
 
 	beaconState, err := rs.db.HeadState(ctx)
 	if err != nil {
 		log.Errorf("Failed to get beacon state: %v", err)
-		return [32]byte{}, false, err
+		return nil, nil, false, err
 	}
 
-	span.AddAttributes(
-		trace.Int64Attribute("block.Slot", int64(block.Slot)),
```

Comment:
Revert this change.

Also, while you are here, can you change both of these values to be `x - params.BeaconConfig().GenesisSlot`? They overflow currently